client/src/common/components/Members/components/MemberDetailsHeader.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Avatar from 'common/components/Avatar';
+import MemberRole from 'common/components/Members/components/MemberRole';
+
+const MemberDetailsHeader = ({
+  avatarUrl,
+  displayName,
+  isCohortList,
+  isGuest,
+  isMember,
+  isOwner
+}) => (
+  <header className="member-details-header">
+    <div className="member-details-header__avatar">
+      <Avatar
+        avatarUrl={avatarUrl}
+        className="member-details-header__image"
+        name={displayName}
+      />
+    </div>
+    <div>
+      <h3 className="member-details-header__name">{displayName}</h3>
+      <MemberRole
```

Comment:
Consider passing this as a `role` prop - it will remove the dependency on the `isCohortList, isGuest, isMember, isOwner`  prop set.

```javascript
const role = <MemberRole 
  isCohortList={isCohortList}
  isGuest={isGuest}
  isMember={isMember}
  isOwner={isOwner}
/>

return <MemberDetailsHeader
  avatarUrl={avatarUrl}
  displayName={displayName}
  role={role}
/>
```